src/Footer/Footer.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Footer from './Footer';
+
+const setup = () => {
+  const component = shallow(<Footer />);
```

Comment:
`const setup = () => shallow(<Footer />);`

usually this function is called like `renderFooter`

Reply:
Implemented